tests/stubtest_allowlists/py3_common.txt

```diff
@@ -272,3 +272,437 @@ os.O_[A-Z_]+
+email.quoprimime
+email.utils
+hmac.HMAC.blocksize
+importlib.abc.Finder.find_module
+importlib.machinery.WindowsRegistryFinder.DEBUG_BUILD
+importlib.machinery.WindowsRegistryFinder.REGISTRY_KEY
+importlib.machinery.WindowsRegistryFinder.REGISTRY_KEY_DEBUG
+inspect.Signature.from_builtin
+inspect.Signature.from_function
+lib2to3.pgen2.grammar.Grammar.loads
+logging.config.BaseConfigurator
+logging.config.ConvertingDict
+logging.config.ConvertingList
+logging.config.ConvertingMixin
+logging.config.ConvertingTuple
+logging.config.DictConfigurator
+logging.config.dictConfigClass
+mimetypes.MimeTypes.add_type
+modulefinder.test
+multiprocessing.dummy.Pipe
+multiprocessing.managers.Server.accepter
+multiprocessing.managers.Server.create
+multiprocessing.managers.Server.debug_info
+multiprocessing.managers.Server.decref
+multiprocessing.managers.Server.dummy
+multiprocessing.managers.Server.fallback_getvalue
+multiprocessing.managers.Server.fallback_mapping
+multiprocessing.managers.Server.fallback_repr
+multiprocessing.managers.Server.fallback_str
+multiprocessing.managers.Server.get_methods
+multiprocessing.managers.Server.handle_request
+multiprocessing.managers.Server.incref
+multiprocessing.managers.Server.number_of_objects
+multiprocessing.managers.Server.public
+multiprocessing.managers.Server.serve_client
+multiprocessing.managers.Server.shutdown
+multiprocessing.managers.SyncManager.Barrier
+multiprocessing.managers.SyncManager.JoinableQueue
+multiprocessing.managers.SyncManager.Pool
+multiprocessing.pool.Pool.Process
+multiprocessing.pool.ThreadPool.Process
+multiprocessing.reducer
+multiprocessing.synchronize.Semaphore.get_value
+os.stat_result.n_fields
+os.stat_result.n_sequence_fields
+os.stat_result.n_unnamed_fields
+os.terminal_size.n_fields
+os.terminal_size.n_sequence_fields
+os.terminal_size.n_unnamed_fields
+os.times_result.n_fields
+os.times_result.n_sequence_fields
+os.times_result.n_unnamed_fields
+pickle.Pickler.memo
+pickle.Unpickler.memo
+pipes.Template.makepipeline
+pipes.Template.open_r
+pipes.Template.open_w
+pkgutil.ImpImporter.find_module
+pkgutil.ImpImporter.iter_modules
+pkgutil.ImpLoader.code
+pkgutil.ImpLoader.get_code
+pkgutil.ImpLoader.get_data
+pkgutil.ImpLoader.get_filename
+pkgutil.ImpLoader.get_source
+pkgutil.ImpLoader.is_package
+pkgutil.ImpLoader.load_module
+pkgutil.ImpLoader.source
+profile.Profile.dispatch
+profile.Profile.fake_code
+profile.Profile.fake_frame
+profile.Profile.trace_dispatch
+profile.Profile.trace_dispatch_c_call
+profile.Profile.trace_dispatch_call
+profile.Profile.trace_dispatch_exception
+profile.Profile.trace_dispatch_i
+profile.Profile.trace_dispatch_l
+profile.Profile.trace_dispatch_mac
+profile.Profile.trace_dispatch_return
+pydoc.Helper.symbol
+pydoc.Helper.symbols_
+pydoc.Helper.topic
+random.Random.VERSION
+rlcompleter.Completer.attr_matches
+rlcompleter.Completer.global_matches
+sunau.Au_read.initfp
+sunau.Au_write.initfp
+threading.Event.isSet
```

Comment:
This is a deprecated alias to another method (https://docs.python.org/3/library/threading.html#threading.Event.is_set), so shouldn't ever be added to the stub in my opinion -- I'd put it in a "Won't fix" section

Reply:
threading has other aliases too. We can remove them, or add `isSet`, but either way it's beyond the scope of this PR.

Reply:
Fair!